docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc

```diff
@@ -678,47 +710,50 @@ mill clean "foo.{compile,run}"
 mill clean foo.compile foo.run
 mill clean _.compile
 mill clean __.compile
-```
+----
 
-### Search for dependency updates
+=== Search for dependency updates
 
-```bash
+[source,bash]
+----
 $ mill mill.scalalib.Dependency/showUpdates
-```
+----
 
 Mill can search for updated versions of your project's dependencies, if available from your project's configured
 repositories. Note that it uses heuristics based on common versionning schemes, so it may not work as expected for
 dependencies with particularly weird version numbers.
 
 Current limitations:
 
-- Only works for `JavaModule`s (including `ScalaModule`s,
-  `CrossScalaModule`s, etc.) and Maven repositories.
-- Always applies to all modules in the build.
-- Doesn't apply to `$ivy` dependencies used in the build definition itself.
+* Only works for `JavaModule`s (including `ScalaModule`s,
```

Comment:
These need to be double backticks: "``JavaModule``s" etc.  
 https://docs.antora.org/antora/2.3/asciidoc/monospace/#inline-monospace-syntax

Reply:
These are usually plurals, just Ctrl+F for `s 😄